Files/UserControls/Widgets/LibraryCards.xaml.cs

```diff
@@ -166,7 +164,7 @@ private async void DeleteLibrary_Click(object sender, RoutedEventArgs e)
 
         private async Task<byte[]> GetIcon(string path)
         {
-            return await LoadIconOverlay(path, 48u);
+            return (await FileThumbnailHelper.LoadIconOverlayAsync(path, 48u)).IconData;
```

Comment:
Since here you don't care about the overlay, if you wish there is also the `FileThumbnailHelper.LoadIconWithoutOverlayAsync` which only returns the icon

Reply:
Thanks